src/index.js

```diff
@@ -62,7 +62,15 @@ export default function loader(content) {
     );
   }
 
+  // Memcache output paths (files)
+  const cache = new Set();
+
   if (options.emitFile === undefined || options.emitFile) {
+    // Don't emit files again if cached
+    if (cache.has(outputPath)) return null;
+
+    cache.add(outputPath);
```

Comment:
What about if file was changed? Not emitted new?

Reply:
Yep, with the current solution you need to restart, this is a trivial guard atm to avoid remitting 'static assets' constantly when changing e.g CSS files. Should we really hash them ? Is there a low overhead solution to do so ?

Reply:
@michael-ciniawsky i think `hash` is best solution for this, or maybe we can use `memory-fs` which build-in `webpack` for this purpose? Need to ask @sokra about using `memory-fs` (or other cache mechanism) into `loader`

Reply:
I take a look while writing test, if there is a hash available somewhere and try to use it as the identifier

Reply:
Curious if this is going to be an net gain or a net loss performance wise considering we would have to introduce hashing into the mix once you start throwing entire applications at it.

Reply:
Yep, if their is a hash on the `module` `{Object}` anyways (need to check it) or another prop I can use as identifier, fine. But pulling e.g `crypto` in and generate a hash just for this trivial cache, it could likely, compared to just remit the file(s), turn out to nearly have the same perf footprint then and is therefore not really worth the effort. In case their isn't any useable identifier it could be hidden behind a check for `watch` mode or a opt-in e.g (`options.cache`) with docs that file content changes need a restart when caching && `webpack -w` is used. Albeit assets are less likely to change during one session anyways. I will tackle this after all the defaults-test stuff :)